website/src/components/common/ErrorBox.tsx

```diff
@@ -12,13 +12,24 @@ interface Props {
 const ErrorBox: React.FC<Props> = ({ title, children, level = 'error' }) => {
     const alertClass = `my-8 alert ${level === 'error' ? 'alert-error' : 'alert-warning'}`;
 
+    const ContentWithStyledLinks: React.FC<{ children: React.ReactNode }> = ({ children }) => {
```

Comment:
I'm not sure if this is a good pattern but we can come back to it later

Reply:
It's the only one I managed to get working that didn't involve global CSS stuff, I got this from Claude. Feel free to refactor to something better, I fully admit this looks not ideal. But the default of not differentiating links was not good.

Reply:
Made an issue: https://github.com/loculus-project/loculus/issues/3025

Reply:
I don't think there's a problem with global CSS stuff - one is compensating for other global CSS stuff